README.adoc

```diff
@@ -126,11 +126,10 @@ inventory/Dockerfile
 include::finish/inventory/Dockerfile[]
 ----
 
-The [hotspot=from file=0]`FROM` instruction initializes a new build stage, which indicates the parent image of the built image. If you don't need a parent image, then you can use `FROM scratch`, which makes your image a
-base image. 
+The [hotspot=from file=0]`FROM` instruction initializes a new build stage, which indicates the parent image of the built image. If you don't need a parent image, then you can use `FROM scratch`, which makes your image a base image. 
 
-In this case, you're using the `open-liberty` image as your parent
-image, which comes with the Open Liberty runtime by default. You can find all different official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^]. To pull a different open-liberty image, like `kernel`, define the `FROM` instruction as `FROM open-liberty:kernel`. If you build an image for production, it is recommended to use the `openliberty/open-liberty:kernel-java8-openj9-ubi` image.
+In this case, you are using the recommended production image, 
+`openliberty/open-liberty:kernel-java8-openj9-ubi`, as your parent image. If you would not like to have any additional runtime features for your `kernel` image, define the `FROM` instruction as `FROM open-liberty:kernel`. To use the default image that comes with the Open Liberty runtime, simply define the `FROM` instruction as `FROM open-liberty`. You can find all the different official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^].
```

Comment:
Suggested update for this paragraph: 
>In this case, you're using the recommended production image, 
`openliberty/open-liberty:kernel-java8-openj9-ubi`, as your parent image. If you don't want any additional runtime features for your `kernel` image, define the `FROM` instruction as `FROM open-liberty:kernel`. To use the default image that comes with the Open Liberty runtime, define the `FROM` instruction as `FROM open-liberty`. You can find all the official images at https://hub.docker.com/_/open-liberty[open-liberty Docker Hub^].

Reply:
Thanks @Charlotte-Holt. Updated the README.